.github/workflows/commitlint.yml

```
@@ -0,0 +1,30 @@
+name: CI
+
+on: [pull_request]
```

The format is slightly different, see [this example](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches).
```suggestion
on:
  pull_request:
    branches:
      - main
```